<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>Chromatin States</title>
  <style>
    body {
      margin: 0 auto;
      hyphens: auto;
      overflow-wrap: break-word;
      text-rendering: optimizeLegibility;
      font-kerning: normal;
    }
    p {
      margin: 1em 0;
    }
    img {
      max-width: 100%;
    }
    svg {
      height: auto;
      max-width: 100%;
    }
    h1, h2, h3, h4, h5, h6 {
      margin-top: 1.4em;
    }
    h5, h6 {
      font-size: 1em;
      font-style: italic;
    }
    h6 {
      font-weight: normal;
    }
    ol, ul {
      padding-left: 1.7em;
      margin-top: 1em;
    }
    li > ol, li > ul {
      margin-top: 0;
    }
    code {
      font-family: Menlo, Monaco, Consolas, 'Lucida Console', monospace;
      font-size: 85%;
      margin: 0;
      hyphens: manual;
    }
    pre {
      margin: 1em 0;
      overflow: auto;
    }
    pre code {
      padding: 0;
      overflow: visible;
      overflow-wrap: normal;
    }
    .sourceCode {
     background-color: transparent;
     overflow: visible;
    }
    hr {
      background-color: #1a1a1a;
      border: none;
      height: 1px;
      margin: 1em 0;
    }
    table {
      margin: 1em 0;
      border-collapse: collapse;
      width: 100%;
      overflow-x: auto;
      display: block;
      font-variant-numeric: lining-nums tabular-nums;
    }
    table caption {
      margin-bottom: 0.75em;
    }
    tbody {
      margin-top: 0.5em;
    }
    th {
      padding: 0.25em 0.5em 0.25em 0.5em;
    }
    td {
      padding: 0.125em 0.5em 0.25em 0.5em;
    }
    header {
      margin-bottom: 4em;
      text-align: center;
    }
    #TOC li {
      list-style: none;
    }
    #TOC ul {
      padding-left: 1.3em;
    }
    #TOC > ul {
      padding-left: 0;
    }
    #TOC a:not(:hover) {
      text-decoration: none;
    }
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    /* The extra [class] is a hack that increases specificity enough to
       override a similar rule in reveal.js */
    ul.task-list[class]{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      font-size: inherit;
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
    /* CSS for citations */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
      margin-bottom: 0em;
    }
    .hanging-indent div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }  </style>
</head>
<body>
<header id="title-block-header">
<h1 class="title">Chromatin States</h1>
</header>
<p>Chromatin states were generated with ChromHMM <span class="citation"
data-cites="ernstChromHMMAutomatingChromatin2012">(Ernst &amp; Kellis,
2012)</span>. Figure 1 below shows the chromatin state names and colors
followed by the emissions profiles, relative enrichment over gene
features, genomic coverage, gene counts associated with each state, and
expression values associated with each state. See paper methods for
details on how these were calculated.</p>
<figure>
<img src="https://seqcode.github.io/pfal-chrom-states-trackhub/trackhub/GCF_000002765.5/docs/mtp_emissions_editedAtac.png"
alt="Emission probabilities for P. falciparum chromatin states" />
</figure>
<p><strong>Figure 1.</strong> Chromatin states in the Plasmodium IDC and
associated feature enrichments. <strong>A)</strong> Hidden Markov Model
emission probabilities discovered by ChromHMM. <strong>B)</strong>
Relative enrichment of each state at a selection of genomic features.
<strong>C)</strong> The genomic coverage of each state at each
timepoint. R= ring stage (10hpi), T= trophozoites (30hpi), and S=
schizont stage (40hpi). <strong>D)</strong> The number of genes that
have each state located at the start of the coding sequence at each
timepoint. <strong>E)</strong> Z-score values for the average gene
expression value per chromatin state.</p>
<h3 class="unnumbered" id="references">References</h3>
<div id="refs" class="references csl-bib-body hanging-indent"
data-entry-spacing="0" data-line-spacing="2" role="list">
<div id="ref-ernstChromHMMAutomatingChromatin2012" class="csl-entry"
role="listitem">
Ernst, J., &amp; Kellis, M. (2012). <span>ChromHMM</span>: Automating
chromatin state discovery and characterization. <em>Nature Methods</em>,
<em>9</em>(3), 215â€“216. <a
href="https://doi.org/10.1038/nmeth.1906">https://doi.org/10.1038/nmeth.1906</a>
</div>
</div>
</body>
</html>
